pubsub/src/main_sub.cpp

```diff
@@ -49,28 +48,28 @@ void signalHandler(int signal) {
 class CustomListener : public UListener {
 
     public:
-        /* in this example the same onReceive callback implementation is used to receive
-         * the three different messages , each message is differntiated by the URI
-         * it is possible to provide a different onReceive callback for each topic */
-        UStatus onReceive(const UUri& uri,
-                          const UPayload& payload,
-                          const UAttributes& attributes) const override {
-                                
-            if (TIME_URI_STRING == LongUriSerializer::serialize(uri)) {
+
```

Comment:
can you please return the comment?

Reply:
The comment was removed in one of the commits from #5 and not this PR. See above comment with link to diff view for only this PR.

Reply:
@stevenhartley - Can we resolve this conversation?